eahub/localgroups/views.py

```diff
@@ -18,6 +18,8 @@
 from .forms import LocalGroupForm
 from .models import LocalGroup
 from ..profiles.models import Profile
+from ..profiles.views import ReportAbuse
```

Comment:
Seems like this is specific to neither profiles nor local groups, so should be in some common directory like `base` where they can both import it from.

Reply:
done